src/datasources/cache/cache.first.data.source.ts

```diff
@@ -1,11 +1,12 @@
+import { Inject, Injectable } from '@nestjs/common';
+import * as winston from 'winston';
 import { NetworkRequest } from '../network/entities/network.request.entity';
-import { CacheService, ICacheService } from './cache.service.interface';
 import {
   INetworkService,
   NetworkService,
 } from '../network/network.service.interface';
-import { Inject, Injectable } from '@nestjs/common';
-import * as winston from 'winston';
+import { CacheService, ICacheService } from './cache.service.interface';
+import { CacheDir } from './entities/cache-dir.entity';
```

Comment:
We need to double check the import rules else the import order constantly changes between PRs.

Reply:
I'm checking solution to this.

Reply:
Yes, sorry, I reverted the imports order changes in [393ea7b](https://github.com/5afe/safe-client-gateway-nest/pull/285/commits/393ea7b48995c3432938360953ca1ee336732664). This is an automatic feature of my IDE (VS Code), but we could delegate this utility into a prettier/eslint plugin (I found [organize-imports](https://www.npmjs.com/package/prettier-plugin-organize-imports) and [eslint-plugin-import](https://github.com/import-js/eslint-plugin-import)) to have a uniform behavior between all the developers environments.